# enables the combination of face detections with the openni people segmentation
# bool
use_people_segmentation: false

# timespan during which a face is preserved in the list of tracked faces although it is currently not visible
# double
face_redetection_time: 4.0

# the minimum area inside the face rectangle found in the color image that has to be covered with positive people segmentation results (from openni_tracker)
# double
min_segmented_people_ratio_face: 0.7

# the minimum area inside the head rectangle found in the depth image that has to be covered with positive people segmentation results (from openni_tracker)
# double
min_segmented_people_ratio_head: 0.2

# maximum tracking distance for a face (in meters), i.e. a face can move this distance between two images and can still be tracked
# double
tracking_range_m: 0.2

# face identification score decay rate (0 < x < 1), i.e. the score for each label at a certain detection location is multiplied by this factor
# double
face_identification_score_decay_rate: 0.9

# minimum face identification score to publish (0 <= x < max_score), i.e. this score must be exceeded by a label at a detection location before the person detection is published (higher values increase robustness against short misdetections, but consider the maximum possible score max_score w.r.t. the face_identification_score_decay_rate: new_score = (old_score+1)*face_identification_score_decay_rate --> max_score = face_identification_score_decay_rate/(1-face_identification_score_decay_rate))
# double
min_face_identification_score_to_publish: 3.0

# if this is true, the unknown label will be assigned for the identification of a person if it has the highest score, otherwise, the last detection of a name will display as label even if there has been a detection of Unknown recently for that face (i.e. if true: face names will display only while they are certain, if false: face names will always display after they have been certain once)
# bool
fall_back_to_unknown_identification: true

# enables some debug outputs
# bool
debug: true

# display timing information
# bool
display_timing: false
